addons/l10n_fr_pdp_reports/data/pdp_cron.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="ir_cron_l10n_fr_pdp_generate_flows" model="ir.cron">
+            <field name="name">PDP Generate Flux 10 Flows</field>
+            <field name="model_id" ref="model_l10n_fr_pdp_flow_aggregator"/>
+            <field name="state">code</field>
+            <field name="code">model._cron_generate_daily_flows()</field>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field name="doall">False</field>
```

Comment:
FYI in the forward port some field there will be gone like the doall and numbercall :smile: